frontend/src/components/InfoPanel.jsx

```diff
@@ -238,6 +236,14 @@ class InfoPanel extends React.Component {
                       this.state.totalCasesCounterfactual
                     }
                     totalCasesReal={this.state.totalCasesReal}
+                    shiftFirstRestrictions={
+                      new Date(this.state.dateFirstRestrictionsReal) -
+                      new Date(this.state.dateFirstRestrictionsCounterfactual)
+                    }
```

Comment:
Should this be a member function that we call instead of something that happens in place?

Reply:
Done.